aiidalab_widgets_base/structures.py

```diff
@@ -735,7 +735,13 @@ def mod_element(self, _=None):
 
         if self.ligand.value == 0:
             for idx in self.selection:
-                atoms[idx].symbol = self.element.value
+                new = Atom(self.element.value)
+                atoms[idx].mass = new.mass
+                atoms[idx].magmom = new.magmom
+                atoms[idx].momentum = new.momentum
+                atoms[idx].symbol = new.symbol
+                atoms[idx].tag = new.tag
+                atoms[idx].charge = new.charge
```

Comment:
```suggestion
                atoms[idx] = Atom(self.element.value)
```
would this work as well?

Reply:
apparently not, I've made [an issue](https://gitlab.com/ase/ase/-/issues/667) to the ASE, let's see what they reply.

Reply:
According to what was proposed [here](https://gitlab.com/ase/ase/-/issues/667), the right way should be:
```suggestion
                new = Atoms([Atom(self.element.value)])
                for key in atoms.arrays.keys():
                    atoms.arrays[key][idx] = new.arrays[key][0]
```

Reply:
@cpignedoli I realized that the latter change I proposed is not really good, because if `new` contains arrays that `atoms` object doesn't - they aren't going to be added. So I am going to merge your solution for the time being.